vars/main.yml

```diff
@@ -121,7 +121,7 @@ postgresql_parameters:
   - { option: "max_files_per_process",               value: "4096"    }
   - { option: "archive_mode",                        value: "on"      }
   - { option: "archive_timeout",                     value: "1800s"   }
-  - { option: "archive_command",                     value: "cd ."    } # not doing anything yet with WAL-s
+  - { option: "archive_command",                     value: "wal-g wal-push '%p'"    } # not doing anything yet with WAL-s
```

Comment:
You should not override the archive_command parameter. It must be specified manually. By default, leave "cd."

Reply:
Another variant:
leave prepared parameter, but comment out by default, example:
```
  - { option: "archive_command",                     value: "cd ."    } # not doing anything yet with WAL-s
#  - { option: "archive_command",                     value: "wal-g wal-push '% p'" } # archive WAL-s using WAL-G
```